README.md

```diff
@@ -20,19 +24,14 @@ And then execute:
 
 Or install it yourself as:
 
-    $ gem install rdf-ldp
+    $ gem install rdf-vocab  # (was rdf-ldp)
```

Comment:
Do you mean `include` or `require`?  I think you want `require` here.  See
http://stackoverflow.com/questions/318144/what-is-the-difference-between-include-and-require-in-ruby